commons-lang/src/main/java/org/apache/commons/lang3/mutable/MutableLong.java

```diff
@@ -121,11 +121,12 @@ public void increment() {
      * @return the value associated with the instance before it was incremented
      * @since 3.5
      */
-        public long getAndIncrement() {
-    final long last = value;
+    public long getAndIncrement() {
+      final long last = value;
             value++;
```

Comment:
indentation  for line 126 not fixed

Reply:
Good catch edited my comment!

Reply:
Is this ready to merge?

Reply:
not yet @PentesterPriyanshu needs to fix indentation in line 126 `        value++;`